### PR TITLE
Supporting Neptune in the Databuilder

### PR DESCRIPTION
**Pull request by AndrewCiambrone**

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @friendtocephalopods to take a look

**Comment by feng-tao** [comment]
the rfc lgtm

File: rfcs/000-neptune-support.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+- Feature Name: Amazon Neptune Databuilder support
+- Start Date: 2020-11-10
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
```
Comment (friendtocephalopods) [inline rfcs/000-neptune-support.md:3]: tiny nit, I think we should update this?

**Comment by friendtocephalopods** [comment]
LGTM!

File: rfcs/000-neptune-support.md (hunk 1 above)
Comment (Golodhros) [reply rfcs/000-neptune-support.md:3]: Yep

Comment (Golodhros) [inline rfcs/000-neptune-support.md:3]:
```suggestion
- RFC PR: [amundsen-io/rfcs#13](https://github.com/amundsen-io/rfcs/pull/13)
```

File: rfcs/000-neptune-support.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
```
Comment (Golodhros) [inline rfcs/000-neptune-support.md:4]: Let's create this issue on the issue tracker of Amundsen

**Comment by Golodhros** [comment]
Seems to be acceptance of this proposal, moving it to FCP

**Review by Golodhros — APPROVED** [review]
LGTM

**Pull request merged**